lib/versioning/git/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as generic from '../loose/generic';
```

Comment:
I wanted to find opinions on the versioning scheme for git submodules (I don't think we have any other manager that only supports hashes).

As git submodules only supports hashes, I don't think any of the concepts around versioning really apply. You can't compare them, there's no compatability, etc. My thoughts were basically, if they're different, then we can update.

Reply:
Assuming it's logically possible, I would think that we don't need a concept of "versions" for git submodules and instead treat everything as a "digest update". Does that work?

Reply:
Again, I think git submodules are unique, in that they only have digests, and not explicit versions. Unfortunately all of our logic in Renovate is setup around supporting a version first, and a digest as something extra on top of the version.

This is why I've had to arbitrarily create a version for git submodules, even though it's actually a hash/digest.